source/partials/_top-nav.html.erb

```diff
@@ -0,0 +1,8 @@
+<nav>
+  <ul class="top-nav">
```

Comment:
Consider naming this something that speaks to the purpose of the content, rather than its location. It's more scalable in the long run, since its location could change.